ui/components/AutomationDetail.tsx

```diff
@@ -190,56 +183,81 @@ function AutomationDetail({
     },
   ];
   return (
-    <Flex wide tall column className={className}>
-      <Flex wide end gap="14">
-        {automation?.type === "HelmRelease" ? (
-          <Text capitalize semiBold color="neutral30">
-            Chart Version:{" "}
-            <Text size="large" color="neutral40">
-              {(automation as HelmRelease).helmChart?.version || "-"}
+    <Flex wide tall column className={className} gap="16">
+      <Flex wide>
+        <Flex start>
+          <SyncButton
+            onClick={(opts) => sync.mutateAsync(opts)}
+            loading={sync.isLoading}
+            disabled={automation.suspended}
+          />
+          <Spacer padding="xs" />
+          <Button
+            onClick={() => suspend.mutateAsync()}
+            loading={suspend.isLoading}
+          >
+            {automation.suspended ? "Resume" : "Suspend"}
+          </Button>
+          <CustomActions actions={customActions} />
```

Comment:
I know that there're custom actions between the sync/suspend and applied revision/chart text, but imo it needs a little more space, as right now I only get ~12 characters. Maybe that's enough to determine what you need to out of it?

Reply:
Valid concern and will mentioned in the next iteration to this page